utils_cv/tracking/model.py

```diff
@@ -177,15 +177,45 @@ def fit(
         trainer.set_device(
             self.opt.gpus, self.opt.chunk_sizes, self.opt.device
         )
-
+        
+        # initialize loss vars
+        self.losses_dict = {key: [] for key in ['loss', 'hm_loss', 'wh_loss', 'off_loss', 'id_loss']} 
```

Comment:
[minor] I believe you can use 'defaultdict" here which does not need to be initialized with an empty list.
   from collections import defaultdict
   self.loss_dict = defaultdict()